acceptance/tests/direct_puppet/cached_catalog_remediate_local_drift.rb

```diff
@@ -89,16 +89,18 @@
       assert_equal(r, '0', "reports directory should be empty!")
 
       step "Verify puppet run without drift does not make file request from server"
-      r = on(agent, puppet("agent",
+      on(agent, puppet("agent",
         "--use_cached_catalog",
         "--server", "no_such_host",
         "--report_server", master.hostname,
         "--onetime",
         "--no-daemonize",
         "--detailed-exitcodes",
         "--verbose"
-      )).stderr
-      assert_equal(r, "", "Fail: Did agent try to contact server?")
```

Comment:
I'm wondering if another test left `archive_*` setting behind causing this test to fail.